### ISSUE TITLE
Hide 'Sort By: Score' option when not relevant

### ISSUE THREAD

**Issue by MathiasDPX** [dullage/flatnotes#233]
From Last Modified to Score
![Screenshot_20240814-103100](https://github.com/user-attachments/assets/772c137a-7010-4ea6-b136-7a35ce070ef2)

From Title to Score
![Screenshot_20240814-103048](https://github.com/user-attachments/assets/21f4553f-b10f-4473-9c49-4162edd7b6e3)

**Comment by dullage** [comment]
Hmm, what search term did you use?

**Comment by MathiasDPX** [comment]
@dullage #openmc

**Comment by dullage** [comment]
@MathiasDPX - Ah, I see. As it's a tag only search, all results will have the same score, hence what you're seeing. If you include text searching, the sorting should work as expected.

Ideally, I'd hide this option when it isn't relevant. I'll look to add this in the future.

**Comment by dullage** [comment]
> Ooooooooh sorry for the inconvenience

Not at all. It isn't obvious in the UI but should be. I'll leave this open as a reminder for me to do something about it.
